config/services.yaml

```diff
@@ -30,3 +30,11 @@ services:
 
     # add more service definitions when explicit configuration is needed
     # please note that last definitions always *replace* previous ones
+    App\BasicRum\Layers\DataLayer:
+        public: true
+    App\BasicRum\Layers\DataLayer\Query\Runner:
+        calls:
+            - [setConnection, ['@doctrine.orm.entity_manager']]
+        public: true
+    App\BasicRum\DiagramOrchestrator:
+        public: true
```

Comment:
I have to expose these services to public as we need to use them in tests, but i believe it should be a better way to make it. If you know it, please tell me and I will adjust the code accordingly.